### ISSUE TITLE
Does not work with versionized assets

### ISSUE THREAD

**Issue by binarykitchen** [BrowserSync/browser-sync#19]
Today I realized that I cannot use your cool tool at all because all my assets are version-ized with https://github.com/yeoman/grunt-filerev

Each time an asset changes, a new version is generated and the older version becomes obsolete. In other words, I have to hit reload in the browser to load the newer version with a new URL.

This is something your module cannot deal with (yet?). The biggest problem is the HTML snippet pointing to the old asset.

For example we have this:

```
<link rel="stylesheet" href="/styles/site.min.fd094e44.css">
```

When I modify the LESS file, compile it, and grunt-filerev is giving it a new version, then site.min.fd094e44.css is deleted in the styles folder and a new file something like site.min.34a024e41.css is generated. And that's the problem: How can we fix the href attribute to point to the newer version live?

I can imagine a solution: Look up versions via a JSON file

grunt-filerev is already storing versions in a JSON file something like this:

```
{"/styles/site.css":"/styles/site.b0693088.css"}
```

In other words, introduce a new option which watches that JSON file and corrects the href attributes in the HTML when needed before the sync.

What do you think?
